contracts/reflect/tests/integration.rs

```diff
@@ -18,80 +18,48 @@
 //! 4. Anywhere you see query(&deps, ...) you must replace it with query(&mut deps, ...)
 
 use cosmwasm_std::{
-    coin, coins, from_binary, BankMsg, Binary, HandleResponse, HandleResult, HumanAddr,
-    InitResponse, StakingMsg, StdError,
+    coin, coins, from_binary, testing::MockQuerierCustomHandlerResult, to_binary, BankMsg, Binary,
```

Comment:
`MockQuerierCustomHandlerResult`: that is a mouthful. Maybe make it smaller if we need to import it.

Reply:
I don't find a great way to make it smaller without losing important information. If you implement it via a closure, there is no need to import the type: 94f029cfcbdc257c88a9163ac36d3e2bb48d720d